skbio/util/_testing.py

```diff
@@ -94,3 +94,12 @@ def get_data_path(fn, subfolder='data'):
     path = os.path.dirname(os.path.abspath(callers_filename))
     data_path = os.path.join(path, subfolder, fn)
     return data_path
+
+
+def _skip_if_no_matplotlib():
+    """Skip rest of a test if matplotlib is not available."""
+    try:
+        import matplotlib  # noqa
+    except ImportError:
+        import nose
+        raise nose.SkipTest("matplotlib not installed")
```

Comment:
I've pushed some changes to use @ElDeveloper's suggestion to use numpy's decorator.

Though, to answer @ebolyen's comment, I was using the nose's version for consistency (it's exactly the same as unittest's `SkipTest`, but it's easier to import `nose` and know that everything is there than to remember whether something is in unittest or not).